Bindings/RadarIO.Xamarin.Android.Binding/Transforms/EnumMethods.xml

```diff
@@ -1,10 +1,10 @@
 ﻿<enum-method-mappings>
   <!--
   This example changes the Java method:
-    android.support.v4.app.Fragment.SavedState.writeToParcel (int flags)
+    android.support.v4.app.Fragment.SavedState.writeToParcel (int flags
```

Comment:
Now I think that a rogue search-and-replace must have dropped all of these punctuation marks at the end of lines.